js/src/forum/components/ConfirmModal.js

```diff
@@ -0,0 +1,70 @@
+import Modal from "../../common/components/Modal";
+import Button from "../../common/components/Button";
+
+/**
+ * A basic confirm modal.
+ */
+export default class ConfirmModal extends Modal {
+  /**
+   * Initialize the modal.
+   * @param {String} title Modal title
+   * @param {String} message Message, shown above buttons
+   * @param {String} positiveButton Text to show on positive button
+   * @param {String} negativeButton Text to show on negative button
+   * @param {Function} callback Callback called when 
+   */
+  init(title, message, positiveButton, negativeButton, callback) {
+    super.init();
+
+    this.titleProp = title;
+    this.message = message;
+    this.positiveButton = positiveButton;
+    this.negativeButton = negativeButton;
+    this.positiveCallback = callback;
+    this.loading = false;
+  }
+
+  title() {
+    return this.titleProp;
+  }
+
+  className() {
+    return 'ConfirmModal Modal--small';
+  }
```

Comment:
A question: should I add the `ConfirmModal` class? It doesn't have any additional less code.

Reply:
Yes it should have that class. Even though we don't run changes it on, doesn't mean webmasters won't 👍